Vinyl/TurntableConfiguration.swift

```diff
@@ -19,11 +19,17 @@ public enum RecordingMode {
     case missingVinyl(recordingPath: String?)
 }
 
+public enum RecordingStrategy {
+    case always
+    case nonEmpty
+}
+
 public struct TurntableConfiguration {
     
     public let matchingStrategy: MatchingStrategy
     public let recordingMode: RecordingMode
-    
+    public let recordingStrategy: RecordingStrategy
```

Comment:
Is not possible to add another option to `recordingMode` to indicate that it should always record?

Reply:
I think that should be our default, we should record (persist) always even if there's nothing recorded, I don't think there's a huge relevance in flagging through an error that nothing was recorded.

Reply:
Even better. This means we don't need this extra flag, we just always record if `RecordingMode` is anything except `.none` 👍

Reply:
I'm fine with making it a default behaviour without any flag, if we would add a flag into recording mode it would be a breaking change that's why I went with another property. I'll update implementation.

Reply:
@mluisbrown @dmcrodrigues I just went and removed all functionality that was related to checking if something was recorded so that we always create files, this also resulted in one error case being removed.